ocean_provider/utils/test/test_util.py

```diff
@@ -148,10 +148,11 @@ def test_download_ipfs_file(client):
 
     print(f"got ipfs download url: {download_url}")
     assert download_url and download_url.endswith(f"ipfs/{cid}")
-    response = build_download_response(
-        request, requests_session, download_url, download_url, None
-    )
-    assert response.data, f"got no data {response.data}"
+
+    response = requests_session.get(download_url, stream=True)
```

Comment:
it has to work with build_download_response, or it build_download_response has to be fixed. Please keepp tests on point. We want to test the ipfs download functionality, through provider.